src/change_stream.ts

```diff
@@ -325,10 +388,7 @@ export class ChangeStream extends EventEmitter {
       this.pipeDestinations = [];
     }
     this.pipeDestinations.push(destination);
-    if (!this.cursor) {
-      throw new MongoError('ChangeStream has no cursor, unable to pipe');
-    }
-    return this.cursor.pipe(destination, options);
+    return this.cursorStream.pipe(destination, options);
```

Comment:
If we have `ChangeStreamStream` do we need to override `pipe` anymore? All of these methods that simulate `Readable` go away if we just inherit from `Readable`